projects/angular/src/button/_buttons.clarity.scss

```diff
@@ -160,14 +108,13 @@
 }
 
 @mixin btn-min-max-widths() {
-  min-width: variables.$clr_baselineRem_3;
-  max-width: variables.$clr_baselineRem_15;
+  min-width: #{tokens.$cds-global-space-14};
```

Comment:
SCSS interpolation isn't necessary on property declarations.

Reply:
corrected in commit fabccb1